backend/stats/friend_stats_table.py

```diff
@@ -0,0 +1,8 @@
+class FirendsTable:
```

Comment:
should probably be in a history app together with play history eg. 
and could be HistoryFriendsTable, HistoryPlayTable etc. that gets hit on /histories/<user_id:int>/friends
and /histories/<user_id:int >/plays/, respectively.

Reply:
history has its own endpoint now